src/coreclr/nativeaot/System.Private.CoreLib/src/System/Threading/ThreadPool.Windows.cs

```diff
@@ -13,385 +13,60 @@ namespace System.Threading
     //
```

Comment:
This should be moved/merged with what's under libraries. If you would like to preserve the current native AOT configuration and use Windows threadpool on Windows, it should be done by setting the trimming defaults for native AOT.